src/lib/controller/outgoing_message.ts

```diff
@@ -1,14 +1,25 @@
+import { Association, AssociationGroup } from "zwave-js";
 import { ControllerCommand } from "./command";
 
 export interface ControllerResultTypes {
   [ControllerCommand.beginInclusion]: { success: boolean };
   [ControllerCommand.stopInclusion]: { success: boolean };
   [ControllerCommand.beginExclusion]: { success: boolean };
   [ControllerCommand.stopExclusion]: { success: boolean };
-  [ControllerCommand.removeFailedNode]: {};
+  [ControllerCommand.removeFailedNode]: Record<string, never>;
   [ControllerCommand.replaceFailedNode]: { success: boolean };
   [ControllerCommand.healNode]: { success: boolean };
   [ControllerCommand.beginHealingNetwork]: { success: boolean };
   [ControllerCommand.stopHealingNetwork]: { success: boolean };
   [ControllerCommand.isFailedNode]: { success: boolean };
```

Comment:
@balloob Shouldn't we change the response here as well to i.e. `{ failed: boolean }`?

Reply:
Yes we should.

Reply:
Too many copy pasting 😅